diff --git a/node_modules/@cloudflare/vite-plugin/dist/index.mjs b/node_modules/@cloudflare/vite-plugin/dist/index.mjs
index 144259f..e7f53e4 100644
--- a/node_modules/@cloudflare/vite-plugin/dist/index.mjs
+++ b/node_modules/@cloudflare/vite-plugin/dist/index.mjs
@@ -13968,7 +13968,8 @@ function getLocalDevVarsForPreview(configPath, cloudflareEnv) {
 	const dotDevDotVars = unstable_getVarsForDev(configPath, void 0, {}, cloudflareEnv);
 	const dotDevDotVarsEntries = Array.from(Object.entries(dotDevDotVars));
 	if (dotDevDotVarsEntries.length > 0) return dotDevDotVarsEntries.map(([key, value]) => {
-		return `${key} = "${value?.toString().replaceAll(`"`, `\\"`)}"\n`;
+		const rawValue = value && typeof value === "object" && "value" in value ? value.value : value;
+		return `${key} = "${rawValue?.toString().replaceAll(`"`, `\\"`)}"\n`;
 	}).join("");
 }
 /**
